docs/source/open-software-week/index.md

```diff
@@ -38,7 +39,7 @@ and analyse animal motion from video footage.
 
 :::
 
-:::{grid-item-card} {fas}`microscope;sd-text-primary` Big Microscopy Data
+:::{grid-item-card} {fas}`microscope;sd-text-warning` Big Microscopy Data
```

Comment:
I am confused why this is a warning? (Is it you telling me to get moving on this? 😂 )

Reply:
Never mind - I am un-confused after reading the PR description 🤦 😂 (badly named variable in the colour palette though? A discussion for another day)

Reply:
The pydata-sphinx-theme uses orange as the color for "warnings", which is a sensible choice for most themes. As it so happens, brainglobe is also orange, and I also wanted to use orange for your track. However if we use a different shade of orange to the "warning" shade, the two colours will clash and we get an uncanny valley effect (close but not identical). My solution is to hijack the existing warning colour, guaranteeing that we won't end up with too many shades of orange.

In a similar way, I have "hijacked" the secondary theme colour (purple) for my track. This way track colours just draw from colours that are already in our "palette".